### PR TITLE
changed tea break to 5.30 PM

### PR DESCRIPTION
**Pull request by vetrivel223**
change the tea break to 5.30 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +12,6 @@ and have fun.
 1. Meeting a common point - 6 AM
 2. Bus journey start      - 7 AM
 3. Lunch time             - 12 Noon
-4. Tea break              - 3.30 PM
+4. Tea break              - 4.30 PM
+5. end time               - 5.30 PM
```
Comment (brgcy) [inline README.md:16]: @vetrivel223 This should be capital E

**Review by brgcy — CHANGES_REQUESTED** [review]
Rest looks fine except one small change.

File: README.md (hunk 1 above)
Comment (vetrivel223) [reply README.md:16]: i'll correct sir

**Comment by vetrivel223** [comment]
it is done by mistake

**Pull request closed without merging**